setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: github-bot
+
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | Add the NodeSource Node.js repo
+      command: "bash -c 'curl -sL https://deb.nodesource.com/setup_6.x | bash -'"
+      tags: general
+
+    - name: General | APT Update and upgrade
+      apt: update_cache=yes upgrade=full
+      tags: general
+
+    - name: General | Install baseline packages
+      apt: name={{ item }} state=latest
+      with_items: "{{ baseline_packages }}"
+      tags: general
+
+    - name: General | Install required packages
+      apt: name={{ item }} update_cache=yes state=latest
+      with_items: "{{ packages }}"
+      tags: general
+
+    - name: User | Add {{ server_user }} user
+      user: name="{{ server_user }}" shell=/bin/bash
+      tags: user
+
+    - name: Init | Generate and copy init script
+      template: src=./resources/github-bot.service dest=/lib/systemd/system/github-bot.service
+      tags: init
+
+    - name: Init | Create directory for EnvironmentFile
+      file: path=/home/nodejs/environment state=directory mode=0755 owner=nodejs
+      tags: init
+
+    - name: Init | Generate and copy systemd EnvironmentFile
+      template: src=./resources/environment-file dest=/home/nodejs/environment/github-bot
+      tags: init
+
+    - name: Init | Clone github-bot repo
+      become: yes
+      become_user: "nodejs"
```

Comment:
I would use `{{ server_user }}` here and put it in vars. Although not entirely current, we use `iojs` almost everywhere -- I reckon consistency over correctness is the way here.